src/commands/email/send.js

```diff
@@ -17,9 +19,65 @@ class Send extends BaseCommand {
     await this.promptForSubject();
     await this.promptForText();
     const sendInformation = { to: validToEmail, from: validFromEmail[0], subject: this.subjectLine, text: this.emailText, html: '<p>' + this.emailText + '</p>' };
+    const attachmentVerdict = await this.askAttachment();
+    await this.promptAttachment(attachmentVerdict);
+    if (this.attachment) {
+      const fileContent = this.readFile(this.attachment);
+      const attachment = this.createAttachmentArray(fileContent);
+      sendInformation.attachments = attachment;
+    }
     await this.sendEmail(sendInformation);
   }
 
+  async askAttachment() {
+    this.attachment = this.flags.attachment;
+    if (!this.attachment) {
+      const verdict = await this.inquirer.prompt([
+        {
+          type: 'confirm',
+          name: 'sendAttachment',
+          message: 'Would you like to send an attachment?',
+          default: false
+        }
+      ]);
+      return verdict.sendAttachment;
+    }
+    return false;
+  }
+
+  async promptAttachment(verdict) {
+    if (verdict === true) {
+      const file = await this.inquirer.prompt([
+        {
+          name: 'path',
+          message: Send.flags.attachment.description + ':'
+        }
+      ]);
+      this.attachment = file.path;
+    }
+  }
+
+  readFile(filePath) {
+    this.fileName = path.basename(filePath);
+    try {
+      const coded = fs.readFileSync(filePath, 'base64');
+      return coded;
+    } catch (err) {
+      this.logger.error(filePath + ' file not found.');
```

Comment:
It's not necessarily a file not found issue (might not be readable or something else). Maybe `'Unable to read file: ' + filePath`. And then `this.logger.debug(err)`.